cpp/daal/src/data_management/finiteness_checker.cpp

```diff
@@ -307,58 +499,59 @@ services::Status allValuesAreFiniteImpl(NumericTable & table, bool allowNaN, boo
     const size_t nElements = nRows * nColumns;
     const NTLayout layout  = table.getDataLayout();
 
-    size_t nDataPtrs, nElementsPerPtr;
+    size_t nDataPtrs;
```

Comment:
Delete nDataPtrs and connected code if they aren't used elsewhere.

Reply:
Done